src/Period/SmallPeriodHTMLFormatter.php

```diff
@@ -19,10 +19,10 @@ class SmallPeriodHTMLFormatter implements PeriodFormatterInterface
 
     private $fmtMonth;
 
-    public function __construct()
+    public function __construct($locale)
```

Comment:
Old default value is missing here, this would be a BC break causing issues for existing uses of the library.